assets/src/domain/eventEditor/ui/datetimes/datesList/addNew/ModalBody.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { CSSProperties, useEffect } from 'react';
+import { __ } from '@wordpress/i18n';
+import { FormRenderProps } from 'react-final-form';
+
+import Steps from './Steps';
+import { useIterator } from '@appServices/hooks';
+import { Button } from '@infraUI/inputs';
+import { TicketAssignmentsManager } from '@edtrUI/ticketAssignmentsManager/components';
+import { useDataState } from '@edtrUI/ticketAssignmentsManager/data';
+
+// temporary
+const buttonRowStyle: CSSProperties = { display: 'flex', justifyContent: 'center', marginTop: '2em' };
```

Comment:
Box can be used instead: https://chakra-ui.com/box

Reply:
Yes, styling needs a lot of improvement.